sqlx-sqlite/src/testing/mod.rs

```diff
@@ -15,6 +15,7 @@ impl TestSupport for Sqlite {
     }
 
     fn cleanup_test(db_name: &str) -> BoxFuture<'_, Result<(), Error>> {
+        let db_name = convert_path(db_name);
```

Comment:
It looks like there will be no file deletion:
- in `run_test()` (`sqlx-core/src/testing/mod.rs`) calls the database name transformation
```rs
if let Err(e) = DB::cleanup_test(&DB::db_name(&args)).await {
```
- and here, we're trying to delete it with a new name that won't be found

Reply:
@s373r what about now?

Reply:
@bonega, no more comments! thank you!